tests/bearlib/languages/documentation/documentation_extraction_testdata/default.py

```diff
@@ -37,3 +37,30 @@ def best_docstring(param1, param2):
              Cut to the Next Line.
     """
     return None
+
+def docstring_find(filename):
+    """
+    This is dummy docstring find function!
```

Comment:
You didnt run `pydocstyle` on this, otherwise you would see errors.
(You'll need to ignore errors in pre-existing code; oh how ugly and confusing.)